components/Modal/CreateImageUpload.js

```diff
@@ -447,16 +487,42 @@ class CreateImageUploadModal extends React.Component {
           <span className="pf-l-flex__item pf-c-form__label-text">
             <FormattedMessage defaultMessage="Upload image" />
           </span>
+          <Popover
+            id="popover-help"
+            bodyContent={
+              <FormattedMessage
+                defaultMessage="
+                    Image Builder can upload images you create to a Blob container in {azure}. When the image build is complete 
+                    and the upload action is successful, the image file is available in the Storage account and Blob container that you specified.
+                    "
+                values={{
+                  azure: (
+                    <Button
+                      variant="link"
+                      icon={<ExternalLinkSquareAltIcon />}
+                      iconPosition="right"
+                      isInline
+                      href="https://portal.azure.com/"
+                    >
+                      Microsoft Azure
+                    </Button>
```

Comment:
Per my previous comment, this component should render as an `<a>` html element. If you add the following prop here, that should do it:
`component="a"`

Also per my previous comment, this component is also where you can add `className="pf-icon"` to apply the vertical-align styles to the svg that it includes.

Reply:
I have added your suggested pf-icon css and I updated the buttons to render as an `<a>` element by using `component="a"`